go.mod

```diff
@@ -3,13 +3,7 @@ module github.com/deepmap/oapi-codegen
 go 1.20
 
 require (
-	github.com/apapsch/go-jsonmerge/v2 v2.0.0
 	github.com/getkin/kin-openapi v0.118.0
-	github.com/gin-gonic/gin v1.9.1
-	github.com/go-chi/chi/v5 v5.0.10
-	github.com/gofiber/fiber/v2 v2.49.1
-	github.com/google/uuid v1.3.1
-	github.com/kataras/iris/v12 v12.2.6-0.20230908161203-24ba4e8933b9
 	github.com/labstack/echo/v4 v4.11.1
```

Comment:
@deepmap-marcinr looks like this is pulled in through `pkg/testutil/request_helpers.go` - would you be happy us removing that package too? 

I know there are [a few of the other libraries](https://github.com/search?q=org%3Aoapi-codegen%20testutil&type=code) that depend on it, but I'm not sure how widespread it's used, and it'd be nice to remove this too, as then our direct dependency tree would be:

```
require (
	github.com/getkin/kin-openapi v0.118.0
	github.com/stretchr/testify v1.8.4
	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616
	golang.org/x/text v0.13.0
	golang.org/x/tools v0.12.0
	gopkg.in/yaml.v2 v2.4.0
)
```

Reply:
It's used everywhere in my company, however, we can move it to its own package, yes.

Reply:
Cool, thanks Marcin! I'll try and get the package set up in the next week or so 🤞🏽

Reply:
Raised https://github.com/deepmap/oapi-codegen/pull/1314 for the testutil migration